File: src/main/java/io/github/wntopia/gikipedia/server/domain/auth/dto/response/DataGsmClubSummaryResDto.java
package io.github.wntopia.gikipedia.server.domain.auth.dto.response;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import java.io.Serializable;

@JsonIgnoreProperties(ignoreUnknown = true)
public record DataGsmClubSummaryResDto(
    Long id, String name, String type, String status, Integer foundedYear, Integer abolishedYear)
    implements Serializable {}

File: src/main/kotlin/io/github/wntopia/gikipedia/server/domain/auth/dto/response/DataGsmClubSummaryResDto.kt
package io.github.wntopia.gikipedia.server.domain.auth.dto.response

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
data class DataGsmClubSummaryResDto(
    val id: Long?,
    val name: String?,
    val type: String?,
    val status: String?,
    val foundedYear: Int?,
    val abolishedYear: Int?,
) : Serializable
